dbproject/src/test/java/dbproject/DatabaseUtility.java

```diff
@@ -0,0 +1,41 @@
+package dbproject;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+
+public class DatabaseUtility implements DatabaseAccessor {
+	private Connection con;
+	private String userName = "Alba1020";
+	private String password = "Alba1020!";
+	
+	public DatabaseUtility(String url) {
+		try {
+			this.con = DriverManager.getConnection(url, userName, password);

```

Comment:
This will connect to the database when you create an instance of the class. I suggest lazy-loading instead, that is only connect when you are doing db operations. Then disconnect.